src/include/cursor.h

```diff
@@ -32,6 +32,21 @@
       0                      /* uint32_t flags */                                               \
     }
 
+/*
+ * Block based incremental backup structure. These live in the connection.
+ */
+#define WT_BLKINCR_MAX 2
+struct __wt_blkincr {
+    const char *id;        /* User's name for this backup. */
+    const char *ckpt_name; /* Requires WT-5115. All checkpoints must be this name */
+    void *data;
+/* AUTOMATIC FLAG VALUE GENERATION START */
+#define WT_BLKINCR_INUSE 0x1u /* This entry is active */
+#define WT_BLKINCR_VALID 0x2u /* This entry is valid */
+                              /* AUTOMATIC FLAG VALUE GENERATION STOP */
```

Comment:
Weird indentation?

Reply:
Clang-format does bizarre indenting of that for some reason. @tetsuo-cpp do you know why? It is not limited to this one nor is it consistent among our header files. It appears to align the `STOP` with the flag comments. That is unfortunate, IMO. For instance, in `lsm.h` it is all over the place. Grepping for `AUTOMATIC` shows:
```
lsm.h:/* AUTOMATIC FLAG VALUE GENERATION START */
lsm.h:                                     /* AUTOMATIC FLAG VALUE GENERATION STOP */
lsm.h:/* AUTOMATIC FLAG VALUE GENERATION START */
lsm.h:    /* AUTOMATIC FLAG VALUE GENERATION STOP */
lsm.h:/* AUTOMATIC FLAG VALUE GENERATION START */
lsm.h:/* AUTOMATIC FLAG VALUE GENERATION STOP */
```